boards/arm64/xenvm/xenvm_gicv3_defconfig

```diff
@@ -0,0 +1,25 @@
+CONFIG_SOC_XENVM=y
+CONFIG_BOARD_XENVM=y
+
+# Use GICv3
+CONFIG_XENVM_USE_GIC_V3=y
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+CONFIG_XIP=n
+CONFIG_FLASH_BASE_ADDRESS=0x0
+CONFIG_FLASH_SIZE=0
```

Comment:
why this is needed?

Reply:
Please check my previous answer. I did not add this. This is copied from xenvm_defconfig.

Reply:
right, but is this actually needed? If not you can remove it from both.

Reply:
I checked and I can remove them as 0 is the default value when not specified.